lib/client.js

```diff
@@ -2060,6 +2066,145 @@ MatrixClient.prototype.paginateEventContext = function(eventContext, opts) {
     return promise;
 };
 
+/**
+ * Get an EventTimeline for the given event
+ *
+ * <p>If the room object already has the given event in its store, the
+ * corresponding timeline will be returned. Otherwise, we make a /context
```

Comment:
Nit: Using 'we' for public-facing docs seems weird. I would say "Otherwise, a /context request will be made and used to construct an EventTimeline".